### PR TITLE
Clean up package.json scripts #396

### LINKED ISSUE #396: Clean up package.json scripts
### Currently

There are a lot of ’em!

### Problem

It’s a mess!

### Wanted

Clean it up!

### PR DESCRIPTION
**Pull request by conor-cafferkey-sociomantic**
Closes #396:
- Removed unused visual testing scripts
- Removed linter scripts
- Simplified and renamed all other scripts for clarity

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,42 +18,33 @@
...
-    "test:v:buildFresh": "concurrently \"yarn build\" \"yarn test:v:build\"",
-    "test:v:clean": "yarn loch:clean",
-    "test:v:ref": "visualTest -e dev -c ./ -j src/defaults.json -t reference",
-    "test:v": "visualTest -e dev -c ./ -j src/defaults.json",
-    "test:visuals": "yarn test:v:clean && yarn build && yarn test:v:build && yarn test:v",
+    "bump:minor": "yarn pre-bump && release-it minor",
+    "bump": "yarn pre-bump && release-it",
+    "copy:gh-pages": "copyfiles --flat .lochness/dist/index.html dist/displayComponents.js dist/displayComponentStyles.css docs/ && copyfiles --flat .lochness/dist/assets/* docs/assets/",
+    "copy:sprite": "copyfiles --flat src/Icon/sprite.html dist/sprite.html",
+    "copy:styles": "copyfiles --flat dist/styles.css dist/styles.scss",
```
Comment (mahmoud-zakria-sociomantic) [inline package.json]: I'm not sure, I guess you forgot to add the destination folder which in this case I think `docs`

File: package.json [hunk 2]
```diff
@@ -18,42 +18,33 @@
...
-    "test:v:build": "yarn loch:build",
-    "test:v:buildFresh": "concurrently \"yarn build\" \"yarn test:v:build\"",
-    "test:v:clean": "yarn loch:clean",
-    "test:v:ref": "visualTest -e dev -c ./ -j src/defaults.json -t reference",
-    "test:v": "visualTest -e dev -c ./ -j src/defaults.json",
-    "test:visuals": "yarn test:v:clean && yarn build && yarn test:v:build && yarn test:v",
+    "bump:minor": "yarn pre-bump && release-it minor",
+    "bump": "yarn pre-bump && release-it",
+    "copy:gh-pages": "copyfiles --flat .lochness/dist/index.html dist/displayComponents.js dist/displayComponentStyles.css docs/ && copyfiles --flat .lochness/dist/assets/* docs/assets/",
+    "copy:sprite": "copyfiles --flat src/Icon/sprite.html dist/sprite.html",
```
Comment (mahmoud-zakria-sociomantic) [inline package.json]: Should it be at the end we copy `sprite.html` to `docs` or we keep using the bundled already? but overall renaming other scripts done very well.

File: package.json (hunk 1 above)
Comment (mahmoud-zakria-sociomantic) [inline package.json]: I'm not sure, I guess you forgot to add the destination folder which in this case I think docs

File: package.json (hunk 2 above)
Comment (mahmoud-zakria-sociomantic) [inline package.json]: Should it be at the end we copy sprite.html to docs or we keep using the bundled already? but overall renaming other scripts done very well.

Comment (conor-cafferkey-sociomantic) [reply package.json]: The `copy:sprite` and `copy:styles` are doing what used to be part of `updateStyles` and `updateSprite` (they are not related GitHub Pages)

**Pull request merged**